lib/src/helper/pluto_size_helper.dart

```diff
@@ -196,7 +196,7 @@ class PlutoAutoSizeScale<T> extends PlutoAutoSize<T> {
 
       final minSize = getItemMinSize(item);
 
-      final size = max(minSize, getItemSize(item) * scale);
+      final size = max(minSize, getItemSize(item) * scale + 1);
```

Comment:
I put this `+1`, because it only scales the column and left 1 pixel lasting :cry: With the border lines, it seems ugly.